paramfetch.go

```diff
@@ -153,8 +153,20 @@ func (ft *fetch) maybeFetchAsync(ctx context.Context, name string, info paramFil
 	}()
 }
 
+func hasTrustableExtension(path string) bool {
+	// known extensions include "vk", "srs", and "params"
+	// expected to only treat "params" ext as trustable
+	// by blacklisting "vk" and "srs" to ensure new exts
+	// will not change the behavior
+
+	// we must always check "vk" and "srs" files to ensure
+	// only valid blocks are built upon
+	return !strings.HasSuffix(path, "vk") && !strings.HasSuffix(path, "srs")
```

Comment:
Really sorry I am late here, but this is actually the wrong way around

If in the future we end up with a new critical paramtype, it will become "trusted" implicitly and we are back to square one:
people restart => do not download the new stuff => stop validating => they fork

we should "whitelist" `params` exclusively instead, keeping everything else the same
cc @magik6k

Reply:
I thought about that and I figured "coercing the user into doing the work they deliberately asked not to do" would be the opt-in choice, not an opt-out. But I'm happy to amend this in a following PR.

Reply:
Yes please, let's do it. Sorry once again I'm late on review 😿 
Thanks!